### PR TITLE
Update docs related to selinux

### PR DESCRIPTION
**Pull request by vishesh92**
Documentation updated at these places:
* https://cloudstack-documentation--411.org.readthedocs.build/en/411/developersguide/ansible.html#cloudstack-management-server-service
* https://cloudstack-documentation--411.org.readthedocs.build/en/411/quickinstallationguide/qig.html#selinux
* https://cloudstack-documentation--411.org.readthedocs.build/en/411/installguide/hypervisor/lxc.html#configure-the-security-policies
* https://cloudstack-documentation--411.org.readthedocs.build/en/411/installguide/hypervisor/kvm.html#configure-the-security-policies
* https://cloudstack-documentation--411.org.readthedocs.build/en/411/installguide/management-server/index.html#install-the-database-on-the-management-server-node

----
📚 Documentation preview 📚: https://cloudstack-documentation--411.org.readthedocs.build/en/411/

### CODE REVIEW

**Review by DaanHoogland — APPROVED** [review]

**Comment by sureshanaparti** [comment]
@blueorangutan docbuild

File: source/developersguide/ansible.rst [hunk 1]
```diff
@@ -294,6 +294,10 @@ For the management server role we create a main.yml task like this:
 
 Save this as `/etc/ansible/roles/cloudstack-management/tasks/main.yml`
 
+.. note:: In a production environment, selinux would be set to enforcing
```
Comment (sureshanaparti) [inline source/developersguide/ansible.rst:297]:
```suggestion
.. note:: In a production environment, selinux should be set to enforcing
```

File: source/developersguide/ansible.rst [hunk 2]
```diff
@@ -294,6 +294,10 @@ For the management server role we create a main.yml task like this:
...
+   and the necessary selinux policies would be created to allow the
```
Comment (sureshanaparti) [inline source/developersguide/ansible.rst:298]:
```suggestion
   so that the necessary selinux policies are created to allow the
```

**Comment by sureshanaparti** [comment]
@blueorangutan docbuild

**Review by sureshanaparti — APPROVED** [review]
verified, lgtm

**Comment by andrijapanicsb** [comment]
@vishesh92 - I know this is the OLD one - but just to clarify in my head:

 some of the files have a note:
 
 .. note:: In a production environment, selinux should be set to enforcing
   and the necessary selinux policies are created to allow the
   services to run.
   
   
   Does that mean that we have also changed/updated to actual ACS source code to take care of those policies (create them), or are we leaving this to the operator "go figure it out yourself" kind of thing?

**Comment by vishesh92** [comment]
> @vishesh92 - I know this is the OLD one - but just to clarify in my head:
> 
> some of the files have a note:
> 
> .. note:: In a production environment, selinux should be set to enforcing and the necessary selinux policies are created to allow the services to run.
> 
> Does that mean that we have also changed/updated to actual ACS source code to take care of those policies (create them), or are we leaving this to the operator "go figure it out yourself" kind of thing?

@andrijapanicsb we are leaving this to the operator. The policies will vary depending on their setup and choices of different hypervisors, storage, etc.

**Pull request merged**